addon/converter/fixture-converter.js

```diff
@@ -240,7 +240,7 @@ export default class {
   extractHasMany(fixture, relationship, parentModelName, relationships) {
     let hasManyRecords  = fixture[relationship.key],
         relationshipKey = this.transformRelationshipKey(relationship),
-        isEmbedded      = this.isEmbeddedRelationship(parentModelName, relationshipKey),
+        isEmbedded      = this.isEmbeddedRelationship(parentModelName, relationship.key),
```

Comment:
We need to pass the `key` of the relationship (instead of the transformed one), because otherwise the lookup on the `attrs` has will check for the wrong key. Since the `attrs` hash uses the relationship names as defined in the model, and not the keys which are in the payload...

---

Given

```js
attrs: {
  includedVillains: { embedded: 'always' }
}
```

we want to use `this.isEmbeddedRelationship(parentModelName, "includedVillains")` instead of `this.isEmbeddedRelationship(parentModelName, "included_villains")`.

Reply:
tricky